packages/flutter/lib/src/gestures/tap.dart

```diff
@@ -257,5 +386,7 @@ class TapGestureRecognizer extends PrimaryPointerGestureRecognizer {
     properties.add(FlagProperty('wonArenaForPrimaryPointer', value: _wonArenaForPrimaryPointer, ifTrue: 'won arena'));
     properties.add(DiagnosticsProperty<Offset>('finalPosition', _finalPosition, defaultValue: null));
     properties.add(FlagProperty('sentTapDown', value: _sentTapDown, ifTrue: 'sent tap down'));
+    // TODO(tongmu): Uncomment this line and update related tests.
+    // properties.add(IntProperty('initialButtons', _initialButtons));
```

Comment:
Just delete these two lines: don't check in commented out code:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abandonware